.github/workflows/ci.yml

```diff
@@ -0,0 +1,57 @@
+name: 'CI'
+# **What it does**: Runs test on the CI.
+
+on:
+  push:
+    branches:
+      - master
+      - 'v*'
+  pull_request: {}
+
+jobs:
+  test:
+    name: "Tests"
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14.x
+          cache: yarn
+      - name: Install Dependencies
+        run: yarn install --frozen-lockfile
+      - name: Run Tests
+        run: yarn test:ember
+
+  try-scenarios:
+    name: ${{ matrix.try-scenario }}
+    runs-on: ubuntu-latest
+    needs: "test"
+    timeout-minutes: 10
+
+    strategy:
+      fail-fast: false
+      matrix:
+        try-scenario:
+          - ember-lts-3.20
+          - ember-lts-4.4
+          - ember-release
+          - ember-beta
+          - ember-canary
+          - ember-classic
+          - ember-default-with-jquery
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14.x
+          cache: yarn
+      - name: Install Dependencies
+        run: yarn install --frozen-lockfile
+      - name: Run Tests
+        run: ./node_modules/.bin/ember try:one ${{ matrix.try-scenario }}
```

Comment:
**suggestion (optional)**

I'd change the definition of `test:ember-compatibility` script in `package.json` so that you can write instead,

```yml
run: yarn test:ember-compatibility ${{ matrix.scenario }}
```

For reference, you can check [`ember-container-query`](https://github.com/ijlee2/ember-container-query/blob/2.1.0/package.json#L42), an addon that I maintain.

Reply:
It already contained a [similar command](https://github.com/adopted-ember-addons/ember-launch-darkly/blob/master/package.json#L23) doing a slightly different thing:

```
"test:ember-compatibility": "ember try:each"
```

Let me know if there's a benefit to updating this to reflect the one you suggested

Reply:
My understanding of `try:each` is that, when one runs the command locally, all `ember-try` scenarios will run. This takes a long time to complete so I hadn't understood in the past why one might prefer this command.

In the addon that I maintain, I had re-defined `test:ember-compatibility` to test just 1 scenario. I think this could help a maintainer locally test a failing scenario more easily.

If you prefer to go with the Ember blueprint (often my choice), then I'd leave the script as is (continue to have `try:each`).

Reply:
I'll stick with `try:each` for now and see, otherwise I will update it separately :)